server/config.go

```diff
@@ -70,21 +83,53 @@ func (s *Service) ConfigSection(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	section := httprouter.GetParamFromContext(ctx, "section")
 	var res interface{}
-	switch section {
-	case "auth":
-		res = newAuthConfigResponse(*config)
-	default:
-		Error(w, http.StatusBadRequest, fmt.Sprintf("received unknown section %q", section), s.Logger)
+	res = newLogViewerUIConfigResponse(*config)
+
+	encodeJSON(w, http.StatusOK, res, s.Logger)
+}
+
+// ReplaceLogViewerUIConfig replaces the log viewer UI section of the global application configuration
+func (s *Service) ReplaceLogViewerUIConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	config, err := s.Store.Config(ctx).Get(ctx)
+	if err != nil {
+		Error(w, http.StatusBadRequest, err.Error(), s.Logger)
+		return
+	}
+
+	if config == nil {
```

Comment:
do we need to check this?

Reply:
I'm not sure about this, this was here from before and it seemed safer to leave it

Reply:
not clear to us -- we didn't add this code. it was duplicated from previous code once we split these service functions. thoughts?

Reply:
have to do a bit more investigation, but I think it should be fine to not check it.

Reply:
we've left it for now